dist/setup/index.js

```diff
@@ -13913,20 +13913,18 @@ class MicrosoftDistributions extends base_installer_1.JavaBase {
     }
     getAvailableVersions() {
         return __awaiter(this, void 0, void 0, function* () {
-            // TODO get these dynamically!
-            // We will need Microsoft to add an endpoint where we can query for versions.
             const jdkVersions = [
                 {
-                    version: [17, 0, 1, 12, 1]
+                    version: [17]
                 },
                 {
-                    version: [16, 0, 2, 7, 1]
+                    version: [16]
                 }
             ];
             // M1 is only supported for Java 16 & 17
```

Comment:
@gdams we now provide JDK 11 for M1 as well. I think this block can be simplified.